team_builder_react/src/utilities/API_Interface.js

```diff
@@ -0,0 +1,233 @@
+            {
+                "id": int,
+                "name_last": string,
+                "name_first": string,
+                "skills": string,
+                "bpt_confidence": int,
+                "bpt_delegator": int,
+                "bpt_determination": int,
+                "bpt_disruptor": int,
+                "bpt_independence": int,
+                "bpt_knowledge": int,
+                "bpt_profitability": int,
+                "bpt_relationship": int,
+                "bpt_risk": int,
+                "bpt_selling": int
+            }
+
+             if there was an error in the request, then the Axios error in a Promise.
+*/
+const GetEmployee = async (employeeID) => {
+    // TODO: Test this when the back-end can be queried with values
+    try {
+        const response = await axiosInstance.get('/employees/?id=' + employeeID.toString());
+        return response.data;
+    } catch (error) {
+        return error;
+    }
+}
+
+/*
+    GetEmployeeList is an asynchronous function that takes a list of ID's of the
+        employees whose information you wish to retrieve. This function will make
+        GET requests to the back-end to get each employee's profile/information.
+     
+    parameters: employeeIDList - list of integers where each element is an
+                    integer representing the unique ID that database
+                    uses to identify an employee's record
+
+    returns: if no server errors, a Promise that a list of objects where each 
+                element is an employee's information.
+            [[{
+                "id": int,
+                "name_last": string,
+                "name_first": string,
+                "skills": string,
+                "bpt_confidence": int,
+                "bpt_delegator": int,
+                "bpt_determination": int,
+                "bpt_disruptor": int,
+                "bpt_independence": int,
+                "bpt_knowledge": int,
+                "bpt_profitability": int,
+                "bpt_relationship": int,
+                "bpt_risk": int,
+                "bpt_selling": int
+            }, ...], ...]
+             if there was an error in the request, then the Axios error in a Promise.
+*/
+export const GetEmployeeList = async (employeeIDList) => {
+    // TODO: Test this when the back-end can be queried with values
+    var listOfEmployees = [];
+
+    // Loop through all the id's and get the employee that is associated with them
+    await employeeIDList.forEach(async (id) => {
+        const employee = await GetEmployee(id);
+        
+        // If the response was an error, just return the error
+        if (axios.isAxiosError(employee)){
+            return employee;
+        }
+
+        listOfEmployees.push(employee);
+    });
+    
+    // console.log(listOfEmployees)
+    return listOfEmployees;
```

Comment:
Is the code that's calling this expecting a list or an error, or both?  You're returning either depending on if GetEmployee returns an error.

Reply:
That's a good point. I changed it to where the error is returned in a list as well.